pkg/volume/csi/csi_attacher_test.go

```diff
@@ -1187,9 +1187,13 @@ func TestAttacherMountDevice(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		user, _ := user.Current()
-		if tc.populateDeviceMountPath && user.Uid == "0" {
-			t.Skipf("Skipping intentional failure on existing data when running as root.")
+		user, err := user.Current()
+		if err != nil {
+			t.Logf("Test(s) already running as non-root")
```

Comment:
is this always the case? https://golang.org/pkg/os/user/#Current

Reply:
Good point @BenTheElder. Appended the error to log message and tried with a UID for which there is no user _name_ defined. 
Error output for `UID 2000` with no user `name`:
```
Test(s) already running as non-root: user: unknown userid 2000
```

After doing `make clean`, error output for `UID 65535` with user name `nobody`, got no error. 

So unless for some non-obvious reason if the user _name_ of UID 0 (e.g. root) is not defined in the test environment, we may get this error. But, would suspect this is to be extremely rare. Happy to be proven wrong :) 

Probably makes sense to alter the error log to `"Current user could not be determined: %v", err` for the purposes of this PR with a comment explaining this behavior. Works?

Reply:
maybe roughly: maybe current user could not be determined, assuming non-root?

Reply:
@BenTheElder Fixed it now. Please check